.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- 
+Thanks for contributing a pull request! Please ensure that your PR satisfies the checklist before submitting:
+http://scipy.github.io/devdocs/dev/contributor/development_workflow.html#checklist-before-submitting-a-pr
+
+Please name and describe your PR as you would write a commit message:
+https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html#writing-the-commit-message
+
+Note that we are a team of volunteers; we appreciate your patience during the review process.
+
+Again, thanks for contributing!
+-->
+
+#### Reference issue
+Example: Closes #1234.
```

Comment:
`gh-1234`

Reply:
Also people tend to just fill this and leave out the parts that they don't care. So probably this would ping issue #1234 a lot :) better make that a nonparsable number `<issue number>` etc.

Reply:
Made it gh-WXYZ and put it in a comment.